app/navigation/Navigator.scala

```diff
@@ -147,13 +153,21 @@ class Navigator @Inject() () {
       case Some(_) => HaveYouUsedMethodOneInPastController.onPageLoad(models.NormalMode)
     }
 
-  private def haveYouUsedMethodOneInPastPage(userAnswers: UserAnswers): Call =
-    userAnswers.get(HaveYouUsedMethodOneInPastPage) match {
-      case None        => HaveYouUsedMethodOneInPastController.onPageLoad(models.NormalMode)
-      case Some(true)  => DescribeTheIdenticalGoodsController.onPageLoad(models.NormalMode)
-      case Some(false) =>
+  private def haveYouUsedMethodOneInPastPage(userAnswers: UserAnswers): Call = {
+    val method: Option[ValuationMethod] = userAnswers.get(ValuationMethodPage)
+
+    (userAnswers.get(HaveYouUsedMethodOneInPastPage), method) match {
+      case (None, _)                    => HaveYouUsedMethodOneInPastController.onPageLoad(models.NormalMode)
+      case (Some(true), Some(Method2))  =>
+        DescribeTheIdenticalGoodsController.onPageLoad(models.NormalMode)
+      case (Some(false), Some(Method2)) =>
         WillYouCompareGoodsToIdenticalGoodsController.onPageLoad(models.NormalMode)
+      case (Some(true), Some(Method3))  =>
+        DescribeTheSimilarGoodsController.onPageLoad(models.NormalMode)
+      case (Some(false), Some(Method3)) =>
+        WillYouCompareToSimilarGoodsController.onPageLoad(models.NormalMode)
```

Comment:
Lets break this up into two pages so we don't have an overloaded page & navigator